CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Improve type inference around unsigned integer literals.
```

Comment:
This is specifically around array constructors, right? This should probably be clarified in the changelog.

Reply:
It's not specifically around array constructors - there were distinct problems with the type inference for standalone integer expressions.

Reply:
Ah, I missed the `TypeInferrer::infer` usage in `VariableNameDeclarationImpl`. Makes sense.